src/js/serial.js

```diff
@@ -292,71 +241,68 @@ var serial = {
         });
     },
     getInfo: function (callback) {
-        var chromeType = (this.connectionType == 'serial') ? chrome.serial : chrome.sockets.tcp;
+        const chromeType = (this.connectionType === 'serial') ? chrome.serial : chrome.sockets.tcp;
         chromeType.getInfo(this.connectionId, callback);
     },
     getControlSignals: function (callback) {
-        if (this.connectionType == 'serial') chrome.serial.getControlSignals(this.connectionId, callback);
+        if (this.connectionType === 'serial') {
```

Comment:
For what I can tell `getControlSignals()` and `setControlSignals()` are unused, so this does not really matter. But in general callbacks are often used in JavaScript for function chaining, so implementing a function that accepts a callback but only calls it under certain circumstances is somewhat of an anti-pattern.

These can probably be removed.

Reply:
Removed. Good catch!